pkg/pillar/cmd/zedrouter/appcontainer.go

```diff
@@ -101,3 +112,42 @@ func appContainerGetStats(ipAddr net.IP) (types.AppContainerMetrics, error) {
 	// XXX collect container stats for each container with the docker API endpoint ipAddr
 	return acMetrics, nil
 }
+
+func appChangeContainerStatsACL(newIPAddr, oldIPAddr net.IP) {
+	if oldIPAddr != nil {
+		// remove the previous installed blocking ACL
+		appConfigContainerStatsACL(oldIPAddr, true)
+	}
+	if newIPAddr != nil {
+		// install the App Container blocking ACL
+		appConfigContainerStatsACL(newIPAddr, false)
+	}
+}
+
+func appConfigContainerStatsACL(appIPAddr net.IP, isRemove bool) {
```

Comment:
Would it be feasible to merge this code with other code in acl.go?
It can later become hard to track places from where acl rules get added.

Reply:
sure.